src/content/admin/cleanup.mdx

```diff
@@ -117,3 +220,104 @@ In that case, the Garbage Collector will ignore only those specific resources wh
 You can set the `sleep` and `delete` periods in the [Admin Dashboard](#configuring-the-sleep-and-delete-periods) following the instructions above.
 
 If you are using Okteto Self-Hosted, you can also [configure this in the Helm Chart](self-hosted/helm-configuration.mdx#gc) along with options like a Slack webhook for notifications when resources are scaled to zero.
+
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**Problem: Garbage Collector Not Sleeping Resources**
+
+*Symptoms:* Resources remain active despite exceeding the sleep period
+
+*Solutions:*
+1. **Check GC Status**: Go to **Admin → Garbage Collector** and verify it's enabled
+2. **Verify Timing**: Confirm the sleep period configuration and when the resource was last active
+3. **Check for Activity**: Look for recent deployments, `okteto up` sessions, or kubectl operations
+4. **Review Labels**: Ensure resources don't have `dev.okteto.com/persistent` label
+5. **Check Logs**: Look in **Admin → System Logs** for GC errors or warnings
```

Comment:
This options is not available. The only way of getting the GC logs is getting the GC job logs in the cluster. Not even the support bundle ones.

As the GC are jobs executed every hour (by default), it is most likely that the support bundle doesn't include them